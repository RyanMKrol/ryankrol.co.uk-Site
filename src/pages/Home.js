import React from 'react';

import logo from './../assets/logo.svg';

import './Home.css';

function Home() {
  return (
    <div className="page-body">
      <div className="Home">
        <img src={logo} className="Home-logo" alt="logo" />
        <p>
          Home Page - Under Development!
        </p>
      </div>
    </div>
  );
}

export default Home;
