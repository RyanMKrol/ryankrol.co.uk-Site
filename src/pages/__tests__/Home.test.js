import React from 'react'
import { shallow, mount } from 'enzyme'
import toJson from 'enzyme-to-json'

import Home from './../Home'

// Render Tests

it('renders Home only when rendered', () => {
    const home = shallow(<Home />)
    expect(toJson(home)).toMatchSnapshot()
})

it('does not crash when rendered', () => {
    expect(() => mount(<Home />)).not.toThrow()
})
