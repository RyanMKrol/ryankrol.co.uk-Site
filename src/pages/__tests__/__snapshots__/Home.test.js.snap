// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home only when rendered 1`] = `
<div
  className="page-body"
>
  <div
    className="Home"
  >
    <img
      alt="logo"
      className="Home-logo"
      src="logo.svg"
    />
    <p>
      Home Page - Under Development!
    </p>
  </div>
</div>
`;
